MANIFEST.in

```
@@ -1,2 +1,3 @@
-include README.md LICENSE
-recursive-include pmlb *.py
+include README.md LICENSE 
+include pmlb/all_summary_stats.csv
```

I made this change in the last PR.
```suggestion
include pmlb/all_summary_stats.tsv
```